data/com.endlessm.SystemExternalApps.rules.in

```diff
@@ -0,0 +1,17 @@
+polkit.addRule(function(action, subject) {
+    if (action.id == "com.endlessm.SystemExternalApps.build-install" &&
+        action.lookup("user") == "@helper_user@" && subject.active == true &&
+	subject.local == true) {
+	    if (subject.isInGroup("@privileged_group@"))
+	        return polkit.Result.YES;
+	    else
+	        return polkit.Result.AUTH_ADMIN;
+    }
+    if ((action.id == "org.freedesktop.Flatpak.app-install" ||
```

Comment:
I think this should be split into a separate call to `polkit.addRule` since it's logically separate. I think it should be fine to have 2 calls to `polkit.addRule` in one file.